frontend/src/components/reports/common/PatientStatusReport.js

```diff
@@ -61,6 +62,46 @@ function PatientStatusReport(props) {
     });
   };
 
+  // function to handle csv report start
```

Comment:
this may not be needed . we only need to change the report type param

Reply:
you just need add a link in the report menu in this [component](https://github.com/pascagihozo/OpenELIS-Global-2-UNDP/blob/develop/frontend/src/components/reports/Routine.js#L13)
ie 

 ```
{
      title: <FormattedMessage id="sidenav.title.statusreport.csv" />,
      icon: IbmWatsonDiscovery,
      SideNavMenuItem: [
        {
          link: "/RoutineReport?type=patient&report=CSVPatientStatusReport",
          label: <FormattedMessage id="sidenav.label.statusreport.csv" />,
        },
      ],
    },
```


then add  report component [here ](https://github.com/pascagihozo/OpenELIS-Global-2-UNDP/blob/develop/frontend/src/components/reports/routine/Index.js#L13)

ie 
 {type === "patient" && report === "CSVPatientStatusReport" && (
        <PatientStatusReport
          report={"CSVPatientStatusReport"}
          id={"openreports.patientTestStatus.csv"}
        />
      )}